source/extensions/access_loggers/common/grpc_access_logger.h

```diff
@@ -270,7 +270,7 @@ class GrpcAccessLoggerCache : public Singleton::Instance,
     }
     const auto logger = createLogger(
         config,
-        async_client_manager_.factoryForGrpcService(config.grpc_service(), scope_, false)
+        async_client_manager_.factoryForGrpcService(config.grpc_service(), scope_, true)
```

Comment:
nit: I agree on why we can skip cluster check here. But I think that we should have quick comments about why we can skip cluster check here.